pages/docs/concepts/asyncapi-document/adding_channels.md

```diff
@@ -0,0 +1,180 @@
+---
```

Comment:
The file name is incorrect because we do not use underscored in file names, only dashes. Please fix your file name to be: `adding-channels.md`. ✌🏽

Reply:
Remember: **We observe sentence capitalization in our headers**; please review your doc carefully to fix all your headers accordingly.

Reply:
Nice clear and comprehensive explanation of how to use channels in AsyncAPI, @mhmohona! ✨✨ Let's discuss a few areas to improve: 

### 1. Clarity and Grammar
Some sentences could be restructured for better clarity and understanding. Below is one example to fix; please review your document for more examples to improve in this area.

**Original**: 

"In addition to the term "channel," alternative names can be used to refer to the concept of channels in asynchronous messaging systems."

**Suggestion**:

"The concept of "channels" in asynchronous messaging systems can also be referred to using alternative terms."

### 2. Consistency
Be consistent in how you describe certain aspects. For instance, "Channels and servers have a close relationship in asynchronous messaging systems." could be changed to "In asynchronous messaging systems, channels and servers have a closely linked relationship."

### 3. Redundancy
Avoid redundancy in sentences to keep your document concise.

**Original**:

"In this code code defines a channel `user/signedup` for subscribing to user signup events, and two servers `Server A` and `Server B` with their respective URLs, enabling clients to receive user signup notifications from either server."

**Suggestion**:

"This code defines a channel `user/signedup` and two servers, `Server A` and `Server B`, allowing clients to receive user signup notifications from either server."

### 4. Document Flow
Make sure your document flow is smooth. If you are going to introduce an example after an explanation, it's better to clearly indicate via introductory phrases such as, "Let's look at a code example of...", "For instance, consider the following diagram example..." and so on.

Reply:
Remember: You'll need to update the highlighted nodes in a unique color as we discussed in last call ✌🏽